examples/getstarted/config/environments/development/custom.json

```diff
@@ -1,3 +1,6 @@
 {
-  "myCustomConfiguration": "This configuration is accessible through strapi.config.environments.development.myCustomConfiguration"
+  "myCustomConfiguration": "This configuration is accessible through strapi.config.environments.development.myCustomConfiguration",
+  "public": {
```

Comment:
Can you move to `middleware.json` so the example matches the doc ;)

Reply:
I just removed it, since it is in `application.json` now